.github/workflows/semgrep.yml

```diff
@@ -0,0 +1,26 @@
+Review CI configuration
+Review our semgrep.yml config file.
+
+name: Semgrep
+on:
+  pull_request: {}
+  push:
+    branches:
+      - main
+      - master
+    paths:
+      - .github/workflows/semgrep.yml
+  schedule:
+    - cron: '0 0 * * 0'
+jobs:
+  semgrep:
+    name: Scan
+    runs-on: ubuntu-20.04
+    env:
+      SEMGREP_APP_TOKEN: ${{ secrets.SEMGREP_APP_TOKEN }}
```

Comment:
If this connects to Semgrep App, won’t the whole repo be connected and follow the Semgrep App rule board that is used by all of our org?

Reply:
Yes, there are workarounds though how to apply only specific rules for us.